app/experimenter/experiments/tests/api/v5/test_serializers.py

```diff
@@ -593,6 +594,40 @@ def test_serializer_updates_audience_on_experiment(self):
         )
         self.assertEqual(experiment.total_enrolled_clients, 100)
 
+    def test_serializer_updates_audience_on_experiment_invalid_channels(self):
+        user = UserFactory()
+        experiment = NimbusExperimentFactory(
+            channels=[],
+            application=NimbusExperiment.Application.FENIX,
+            firefox_min_version=None,
+            population_percent=None,
+            proposed_duration=None,
+            proposed_enrollment=None,
+            targeting_config_slug=None,
+            total_enrolled_clients=0,
```

Comment:
Are these all necessary for this test?

Reply:
I wanted to retain them for consistency so it was clear what change causes the error.

Reply:
Ah yep 👍